File: Dot examples/ASP Book Examples/Book Examples/Chapter15/CS/App_Code/Quote.cs
using System;
using System.Collections.Generic;

public class Quote
{
    private List<string> _quotes = new List<string>();

    public string GetQuote()
    {
        Random rnd = new Random();
        return _quotes[rnd.Next(_quotes.Count)];
    }

    public Quote()
    {
        _quotes.Add("All paid jobs absorb and degrade the mind -- Aristotle");
        _quotes.Add("No evil can happen to a good man, either in life or after death -- Plato");
        _quotes.Add("The only good is knowledge and the only evil is ignorance -- Plato");
    }
}

File: Dot examples/ASP Book Examples/Book Examples/Chapter15/VB/App_Code/Quote.vb
Imports System.Collections.Generic

Public Class Quote

    Private _quotes As New List(Of String)

    Public Function GetQuote() As String
        Dim rnd As New Random()
        Return _quotes(rnd.Next(_quotes.Count))
    End Function

    Public Sub New()
        _quotes.Add("All paid jobs absorb and degrade the mind -- Aristotle")
        _quotes.Add("No evil can happen to a good man, either in life or after death -- Plato")
        _quotes.Add("The only good is knowledge and the only evil is ignorance -- Plato")
    End Sub

End Class
